src/index.css

```
@@ -1,19 +1,31 @@
+  --color-primary: #6B705C;
+  --color-accent: #E07A5F;
+  --color-neu-bg: #F9F7F2;
+  --color-workspace-bg: #F9F7F2;
+  --color-sidebar-border: #B7B7A4;
+
+  --font-display: "Plus Jakarta Sans", sans-serif;
+  --font-sans: "Plus Jakarta Sans", sans-serif;
+  --font-story: "Crimson Pro", serif;
+
+  /* Softer shadows for earthy theme */
+  --shadow-neu-flat: 4px 4px 10px rgba(183, 183, 164, 0.25), -4px -4px 10px rgba(255, 255, 255, 0.8);
```

Stylelint (17.2.0)

[error] Expected "0.25" to be "25%" (alpha-value-notation)

(alpha-value-notation)